charts/stable/tdarr/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 2.00.10
 description: Tdarr is a self hosted web-app for automating media library transcode/remux management and making sure your files are exactly how you need them to be in terms of codecs/streams/containers etc.
 name: tdarr
-version: 4.3.1
+version: 4.3.2
```

```suggestion
version: 4.4.0
```
I know it's a small change, but technically it is not a bugfix